src/models/geograph.py

```diff
@@ -86,13 +83,13 @@ def __init__(
             not save the polygonised data.
             tolerance (float, optional): Adds edges between neighbours that are
             at most `tolerance` units apart. Defaults to 0.
+            crs (str, optional): Coordinate reference system to set on the
+            resulting dataframe. Defaults to None.
```

Comment:
Super nit: you probably changed this default later... it defaults to PREFERRED_CRS which is UTM35N (?) at the moment?

Reply:
I'm changing this to remove the default so it's a mandatory argument.